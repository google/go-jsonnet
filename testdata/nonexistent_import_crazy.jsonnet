importstr "ąęółńśćźż \" \' \n\n\t\t"
