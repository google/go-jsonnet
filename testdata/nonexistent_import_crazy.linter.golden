../testdata/nonexistent_import_crazy:1:1-46 couldn't open import "ąęółńśćźż \" ' \n\n\t\t": no match locally or in the Jsonnet library paths

importstr "ąęółńśćźż \" \' \n\n\t\t"


